Files.Launcher/MessageHandlers/FileOperationsHandler.cs

```diff
@@ -348,6 +346,7 @@ private async Task ParseFileOperationAsync(NamedPipeServerStream connection, Dic
                         var moveDestination = ((string)message["destpath"]).Split('|');
                         var operationID = (string)message["operationID"];
                         var overwriteOnMove = (bool)message["overwrite"];
+                        var ownerHwnd = (long)message["HWND"];
```

Comment:
A safer approach:
```suggestion
                        long ownerHwnd = message.Get("HWND", -1L);
```
(`File.Common.Extensions`)

Reply:
Nah if I forgot to pass a parameter I want it to crash so I'll know I have to fix it ;)